rspace/src/test/scala/coop/rchain/rspace/history/HistoryActionTests.scala

```diff
@@ -317,26 +311,25 @@ class HistoryActionTests extends FlatSpec with Matchers with InMemoryHistoryTest
 
 object TestData {
 
-  implicit def toByteVector(bytes: KeyPath): ByteVector = ByteVector(bytes)
+  val _zeros: KeySegment     = KeySegment(List.fill(32)(0).map(_.toByte))
+  val _zerosOnes: KeySegment = KeySegment((List.fill(16)(0) ++ List.fill(16)(1)).map(_.toByte))
+  val _31zeros: KeySegment   = KeySegment(List.fill(31)(0).map(_.toByte))
 
-  val _zeros: KeyPath           = List.fill(32)(0).map(_.toByte)
-  val _zerosOnes: KeyPath       = (List.fill(16)(0) ++ List.fill(16)(1)).map(_.toByte)
-  val _31zeros: KeyPath         = List.fill(31)(0).map(_.toByte)
-  def zerosAnd(i: Int): KeyPath = _31zeros :+ i.toByte
-  def prefixWithZeros(s: String): KeyPath = {
+  def zerosAnd(i: Int): KeySegment = _31zeros :+ i.toByte
+  def prefixWithZeros(s: String): KeySegment = {
     val a = List.fill(32 - s.length)(0).map(_.toByte)
     val b = s.toCharArray.toList.map(_.asDigit).map(_.toByte)
-    a ++ b
+    KeySegment(a ++ b)
   }
```

Comment:
Better to use direct bytes filling instead of `map(._toByte)` conversion: `List.fill(31)(0.toByte)`